src/assets/brand.svg

```diff
@@ -1,4 +1,4 @@
-<svg width="215" height="42" viewBox="0 0 215 42" fill="none" xmlns="http://www.w3.org/2000/svg">
+<svg viewBox="0 0 215 42" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
What is the reason for removing the width and height? It's usually a good idea to set them, but override with CSS, to avoid the 'flash of unstyled SVG' when, before the CSS is loaded, SVGs are rendered as huge images (I've learned about it from this article: https://www.sarasoueidan.com/blog/svg-style-inheritance-and-fousvg/)

Reply:
Having the width and height set in the SVG was preventing us from scaling the SVG up/down. But you're right, I'll see what we can do to prevent the flashing.

Reply:
Ok, so it seems I was wrong, I was just missing the `.w-auto` class in the code where the SVGs were being used. Fixed in 724a8ce.